e2e/specs/clinical-forms.spec.ts

```diff
@@ -119,6 +119,57 @@ test('Fill a clinical form', async ({ page }) => {
   });
 });
 
+test('Clinical form should save inputs even if the encounter time is after the current time', async ({ page }) => {
```

Comment:
Shall we change the title to something like:
"Fill a form with a browser slightly ahead of time"